shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/commitrollback/CommitAndRollbackTestCase.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.integration.transaction.cases.commitrollback;
+
+import lombok.SneakyThrows;
+import org.apache.shardingsphere.integration.transaction.cases.base.BaseTransactionTestCase;
+import org.apache.shardingsphere.integration.transaction.engine.base.TransactionTestCase;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * Commit and rollback transaction integration test.
+ */
+@TransactionTestCase
+public final class CommitAndRollbackTestCase extends BaseTransactionTestCase {
+    
+    public CommitAndRollbackTestCase(final DataSource dataSource) {
```

Comment:
Is it same to `BindingTestCase`?

Reply:
`BindingTestCase` is for testing multi-table associated transaction behavior,`CommitAndRollbackTestCase` is for testing single table transaction behavior.Do I need to merge these two classes?

Reply:
I will change these class names to `MultiTableCommitAndRollbackTestCase` and `SingleTableCommitAndRollbackTestCase`,thank you for your help.